helm/steps-to-deploy/READme.md

```diff
@@ -0,0 +1,35 @@
+## Steps to Deploy ERP into K8s Cluster
+
+1. Clone git repository for helm chart: https://github.com/truthfool/openg2p-erp-docker.git
```

Comment:
Shouldn't include your github repo path.

Reply:
ok i will change it. It was previously there to merge the latest changes